packages/curve-ui-kit/src/lib/model/query/user-address-validation.ts

```diff
@@ -1,18 +1,17 @@
-import type { Address } from 'viem'
 import { enforce, group, test } from 'vest'
 import { createValidationSuite } from '@ui-kit/lib/validation'
 import { isAddress } from 'viem'
 
-type UserAddressParams = { userAddress: Address }
+type UserAddressParams = { userAddress: string }
 
 export const userAddressValidationGroup = ({ userAddress }: UserAddressParams) =>
   group('userAddressValidation', () => {
-    test('userAddress', () => {
-      enforce(userAddress)
-        .message('Address is required')
-        .isNotEmpty()
-        .message('Invalid EVM address')
```

Comment:
In #696 I ended up [filtering](https://github.com/curvefi/curve-frontend/blob/d6a598a3b38f0f7d4ecedf5be71149f0a13f446e/packages/curve-ui-kit/src/lib/validation/lib.ts#L9) fields with 0 errors, the old code seemed to work then :thinking: